src/metrics.jl

```diff
@@ -5,24 +5,107 @@
 #   Metric types
 #
 ###########################################################
+const RealAbstractArray{T <: Real} =  AbstractArray{T}
```

Comment:
Isn't `AbstractArray{<:Real}` short enough?

Reply:
Yes, this was used in the former `wmetrics.jl`, but is unnecessary.